### PR TITLE
Add docs for Node3DGizmo to clarify its link to EditorNode3DGizmo

### LINKED ISSUE #82654: How can I use the Node3DGizmo class?
### Godot version

mono c# 4.1.1

### System information

Deepin os, opengl

### Issue description

How can I use the Node3DGizmo class? Where is the source code of this class? I could not use this feature for some reason.

### Steps to reproduce

my code:
`var gizmo = new Node3DGizmo();`
error:
'Node3DGizmo' does not contain a constructor that takes 0 arguments

### Minimal reproduction project

```
using Godot;
using System;

public partial class Gizmo : Node3D
{
	// Called when the node enters the scene tree for the first time.
	public override void _Ready()
	{
		var gizmo = new Node3DGizmo();
		//AddGizmo(gizmo);
	}

	// Called every frame. 'delta' is the elapsed time since the previous frame.
	public override void _Process(double delta)
	{
	}
}
```

### PR DESCRIPTION
**Pull request by akien-mga**
Fixes #82654.

### CODE REVIEW

File: doc/classes/Node3D.xml [hunk 1]
```diff
@@ -263,7 +265,7 @@
 		<method name="update_gizmos">
 			<return type="void" />
 			<description>
-				Updates all the [Node3DGizmo]s attached to this node.
+				Updates all the Node3D gizmos attached to this node.
```
Comment (YuriSizov) [inline doc/classes/Node3D.xml:268]:
```suggestion
				Updates all the [Node3D] gizmos attached to this node.
```

We don't have good style rules for this, but given that other usages are reference-formatted, I'd keep it consistent.

**Review by YuriSizov — APPROVED** [review]

File: doc/classes/Node3D.xml (hunk 1 above)
Comment (akien-mga) [reply doc/classes/Node3D.xml]: Fixed, together with a few other occurrences (+ grammar errors on those lines).

**Review by YuriSizov — APPROVED** [review]
Still good

**Comment by YuriSizov** [comment]
Thanks!

**Pull request merged**